templates/slider.jsx

```diff
@@ -173,7 +174,7 @@ export default function Slider (props) {
 
         {/* always present start and end notches */}
         <div className="slider__scale-container js-slider-scale" ref={sliderScaleRef}>
-          <div className="slider__scale-notch slider__scale-notch-start"></div>
+          <div className="slider__scale-notch slider__scale-notch-start" />
           {_showScale &&
               <div className="slider__scale-notch-container js-slider-scale-notch-container">
```

Comment:
Very minor and isn't part of this PR but there's an additional indent here.

Reply:
Reduced indent